libs/libfstrm/Makefile

```diff
@@ -0,0 +1,64 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libfstrm
+PKG_RELEASE=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/farsightsec/fstrm.git
+PKG_MIRROR_HASH:=5ba9a313f04f9a9c430efdd452661f8d9a503561541ffa226f8203b253666ab5
+PKG_SOURCE_VERSION:=2559dc42e07df2758b75051b6909e9b26e381fa2
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_INSTALL:=1
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+PKG_BUILD_PARALLEL:=1
+HOST_BUILD_PARALLEL:=1
```

Comment:
IMHO this is  unnecessary since you don't use HostBuild

Reply:
I've removed HOST_BUILD_PARALLEL